File: member/mall-api/src/main/java/com/laf/mall/api/dto/RaffleReward.java
package com.laf.mall.api.dto;

import com.fasterxml.jackson.annotation.JsonView;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class RaffleReward {

    @JsonView(RaffleInfo.RaffleInfoView.class)
    private int trr_id;

    private int raffle_id;

    @JsonView(RaffleInfo.RaffleInfoView.class)
    private String reward_name;

    @JsonView(RaffleInfo.RaffleInfoView.class)
    private int reward_type;

    @JsonView(RaffleInfo.RaffleInfoView.class)
    private String reward_picture;

    private int inventory;

    @JsonView(RaffleInfo.RaffleInfoView.class)
    private double hit_rate;

    @JsonView(RaffleInfo.RaffleInfoView.class)
    private int reward_value;

    @JsonView(RaffleInfo.RaffleInfoView.class)
    private int sort_id;

}

File: member/manager-app/src/main/java/com/laf/manager/dto/RaffleReward.kt
package com.laf.manager.dto

data class RaffleReward(
        var trr_id: Int,
        var raffle_id: Int,
        var reward_name: String,
        var reward_type: Int,
        var reward_picture: String,
        var inventory: Int,
        var hit_rate: Double,
        var reward_value: Int,
        var sort_id: Short
) {
    constructor(
            raffle_id: Int,
            reward_name: String,
            reward_type: Int,
            reward_picture: String,
            inventory: Int,
            hit_rate: Double,
            reward_value: Int,
            sort_id: Short
    ) : this(0, raffle_id, reward_name, reward_type, reward_picture, inventory,
            hit_rate, reward_value, sort_id)

    constructor() : this(0, 0, "", 0, "", 0, 0.0, 0, 0)
}
